src/Helpers/babylon.particleHelper.ts

```diff
@@ -0,0 +1,156 @@
+module BABYLON {
+    /**
+     * Represents all the data needed to create a ParticleSystem.
+     */
+    export interface IParticleSystemData {
+        type: ParticleSystemType;
+        capacity: number;
+        textureFile: string;
+        minEmitBox: { x: number, y: number, z: number };
+        maxEmitBox: { x: number, y: number, z: number };
+        color1: { r: number, g: number, b: number, a: number };
+        color2: { r: number, g: number, b: number, a: number };
+        colorDead: { r: number, g: number, b: number, a: number };
+        minSize: number;
+        maxSize: number;
+        minLifeTime: number;
+        maxLifeTime: number;
+        emitRate: number;
+        blendMode: number;
+        gravity: { x: number, y: number, z: number };
+        direction1: { x: number, y: number, z: number };
+        direction2: { x: number, y: number, z: number };
+        minAngularSpeed: number;
+        maxAngularSpeed: number;
+        minEmitPower: number;
+        maxEmitPower: number;
+        updateSpeed: number;
+    }
+    /**
+     * ParticleSystemType
+     */
+    export enum ParticleSystemType {
```

Comment:
Not needed anymore

Reply:
Mhhh I was thinking this was a good idea to let this with the parse method but yeah if a user made a typo is not our fault ^^